test_collections/matter/sdk_tests/support/python_testing/__init__.py

```diff
@@ -34,6 +34,6 @@
         sdk_mandatory_python_test_collection()
     )
 
-    custom_python_collection: Optional[
-        TestCollectionDeclaration
-    ] = custom_python_test_collection()
+    custom_python_collection: Optional[TestCollectionDeclaration] = (
+        custom_python_test_collection()
+    )
```

Comment:
```suggestion
    custom_python_collection: Optional[
        TestCollectionDeclaration
    ] = custom_python_test_collection()
```
For some reason the `black`, changed this code formatting.

Reply:
@rquidute , I was getting additional CI failures due to black, so I locally ran the reformat... I think the repo is in a bad state. What should we do?